docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -317,25 +317,50 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
+## Take over [filebeat-input-filestream-take-over]
+When `take_over` is enabled, this `filestream` input will take over
+states from the [`log`](/reference/filebeat/filebeat-input-log.md) input
+or other `filestream` inputs. Only states of files being actively
+harvested by this input are taken over.
```

Comment:
I don't it would be entirely clear to the users what states are. It's more an implementation detail.

```suggestion
When `take_over` is enabled, this `filestream` input will take over ingestion of all
files from [`log`](/reference/filebeat/filebeat-input-log.md) or [`filestream`](/reference/filebeat/filebeat-input-filestream.md) inputs that are matched by the `paths` of this `filestream` input.
```

Reply:
I believe they have some idea, but I agree we can word it better.

Your version seems to be missing the fact that only the states of existing files are migrated, I'll re-word it later today.